test/e2e/common/autoscaling_utils.go

```diff
@@ -359,11 +362,63 @@ func (rc *ResourceConsumer) GetReplicas() int {
 	return 0
 }
 
+func (rc *ResourceConsumer) logHPAStatus() {
```

Comment:
This file seems more appropriate in the scalability dir rather than common. Can you move it there?

Reply:
Looks like also sig-cluster-lifecycle for the cluster path